docs/src/main/asciidoc/Getting_Started_Guide.adoc

```diff
@@ -44,22 +44,22 @@ Profile |WildFly {wildflyVersion} Full Platform |WildFly {wildflyVersion} Web Pr
 
 |JSR-372: Jakarta Server Faces 2.3 |X |X |X |X
 
-|JSR-341: Expression Language 3.0 |X |X |X |X
+|JSR-341: Jakarta Expression Language 3.0 |X |X |X |X
```

Comment:
Should we remove the JSR-341 (and other JSR-NNN) references?  

How about the Specification version number?

Reply:
Yes, I guess you are right. I should remove those JSR-NNN references because after all there is no JSR-NNN reference for Jakarta specifications.
And about the specification version number, I'm not completely sure what is the recent specification version for all, which we are going to use in Wildfly 23. That's why I just kept the original one which was there before and replaced it by the Jakarta one. 
and also I think it's good to have the specification version number so that one can check the documents and search the details about it.
@asoldano if you want to add anything.

Reply:
Having spec version is a bit hard to maintain as the spec evolves. We'll need to take care to adjust the spec version to the current one. For Jakarta Batch, it's versoin 1.0 in EE 8, and version 2.0 in EE 9.

Reply:
So, should I remove this spec version?

Reply:
@RanabirChakraborty Removing the spec versions is probably a good idea but it seems out of scope for what you're doing here. You're dividing your work by technology area, while this is something we should discuss and then do as a single piece of work.

Reply:
@RanabirChakraborty , +1 on leaving the removal of the spec version to a future effort.